src/java.base/share/classes/javax/net/ssl/TrustManagerFactory.java

```diff
@@ -75,15 +75,10 @@ public class TrustManagerFactory {
      * implementation-specific default if no such property exists.
      */
     @SuppressWarnings("removal")
-    public static final String getDefaultAlgorithm() {
+    public static String getDefaultAlgorithm() {
         String type;
-        type = AccessController.doPrivileged(new PrivilegedAction<>() {
-            @Override
-            public String run() {
-                return Security.getProperty(
-                    "ssl.TrustManagerFactory.algorithm");
-            }
-        });
+        type = AccessController.doPrivileged((PrivilegedAction<String>) () ->
+            Security.getProperty( "ssl.TrustManagerFactory.algorithm"));
```

Comment:
Another `GetPropertyAction::privilegedGetProperty` candidate.

Reply:
Making the change.